test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -57,6 +57,7 @@ def export_to_onnx(
             f,
             operator_export_type=operator_export_type,
             opset_version=opset_version,
+            verbose=True
```

Comment:
Just making sure: this will go to stdin and will not show if the test runs w/o errors right?

Reply:
It always prints the exported model, failing or not. That is fine on tests pipelines, especially when tracking regressions on a PR